### PR TITLE
Small update

### PR DESCRIPTION
**Pull request by roberts01**
Proposing small updates to the Termux page to reflect recent developments.
Also, maybe its just me, but the commands in the Installation section dont work in Termux.

### CODE REVIEW

**Comment by roberts01** [comment]
If possible, it might be a good idea to put the packages in the dependencies.txt to run from there, and then just put the 'check dependencies' command at the first todo.

File: README.md [hunk 1]
```diff
@@ -65,30 +68,34 @@ The idea is to install "Legacy Version" of WhatsApp on you device via adb and us
...
 
 ![-----------------------------------------------------](https://raw.githubusercontent.com/andreasbm/readme/master/assets/lines/aqua.png)
 
-### Installation
+### Installation (Not working?)
 
 1. Clone the repo
 ```
-git clone https://github.com/yuvrajraghuvanshis/WhatsApp-Key-Database-Extractor.git && cd WhatsApp-Key-Databse-Extractor
+wget https://github.com/yuvrajraghuvanshis/WhatsApp-Key-Database-Extractor && cd WhatsApp-Key-Databse-Extractor
```
Comment (YuvrajRaghuvanshiS) [inline README.md:103]: You need git in termux that's why it didn't work

Comment (roberts01) [reply README.md:103]: ah okay lemme try again

**Review by YuvrajRaghuvanshiS — CHANGES_REQUESTED** [review]
let that be `git clone...` instead of wget.

**Comment by roberts01** [comment]
Changed it back

**Review by YuvrajRaghuvanshiS — APPROVED** [review]

**Pull request merged**